web/src/stream_popover.js

```diff
@@ -635,6 +647,64 @@ export async function build_move_topic_to_stream_popover(
     });
 }
 
+function toggle_stream_info_popover(opts) {
+    const {elt, stream_id} = opts;
```

Comment:
Can you rename it to `toggle_stream_card_popover` and also move `{elt, stream_id}` into the function parameters since the function becomes clearer?

Reply:
I have renamed the function name. However, for the function arguments in the latest commit, I have used only one argument (element) and I have derived the value of stream_id inside the function itself.